frontend/src/components/AddPostForm/AddPostForm.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import { Form, FormTitle, Icon, TitleInput, MessageInput, SubmitInput } from './AddPostForm.styled';
+
+const AddPostForm = () => {
+  const [postState, setPostState] = useState({});
+
+  const handleChange = (event) => {
+    const name = event.target.name;
+    const value = event.target.value;
+    setPostState((prevState) => ({ ...prevState, [name]: value }));
+  };
```

Comment:
Fajny handler! Ja bym co najwyżej zdestrukturyzował event, żeby tylko target wyciagnąć :D

```suggestion
  const handleChange = ({target}) => {
    const name = target.name;
    const value = target.value;
    setPostState((prevState) => ({ ...prevState, [name]: value }));
  };
```

Reply:
Oo, świetne, nie wiedziałem, że tak łatwo można to zrobić.